runtime/doc/map.txt

```diff
@@ -232,8 +232,10 @@ For this reason the following is blocked:
 - Editing another buffer.
 - The |:normal| command.
 - Moving the cursor is allowed, but it is restored afterwards.
+- If the cmdline is changed, the old text and cursor position are restored.
 If you want the mapping to do any of these let the returned characters do
-that.
+that. Alternatively use a |<Cmd>| mapping which doesn't have these
```

Comment:
`<expr>` is not title-case so I guess `<cmd>` should not be either.

Reply:
But `<Cmd>` is a special key, not a flag (yes, confusing to use `<>` for both, but that's vim for you), so it should be title-case.